src/index.js

```diff
@@ -18,15 +18,42 @@ import Player from './Player';
 import Round from './Round';
 import Turn from './Turn';
 
+function createPlayers(Player1, Player2, Player3) {
+  let player1 = new Player(Player1, 1)
+  let player2 = new Player(Player2, 2)
+  let player3 = new Player(Player3, 3)
+  return [player1, player2, player3]
+}
+
+// $(window).load(function() {
+//   // will first fade out the loading animation
+// $('.welcome-message').delay(1500).fadeOut("slow");
+//   // will fade out the whole DIV that covers the website.
+// //$("#preloader").delay(2000).fadeOut("slow");
+// })
 $(document).ready(function() {
-
-  $('#player1-name__submit').click( () => {
- getPlayerName('#player1-name__input', '#player1-name__display')
+  $('.user-name-inputs').hide();
+  $('.puzzle-table').hide();
+  $('.welcome-message').delay(2500).fadeOut("slow");
+  $('.user-name-inputs').append(`
```

Comment:
The DTR is that we are querySelecting with jQuery by #id.

Reply:
Identifier changed to ID